requestStorageAccessForOrigin.bs

```diff
@@ -221,8 +222,43 @@ Modify the [=determine the storage access policy=] algorithm by prepending the f
 
 <h2 id="privacy">Privacy considerations</h2>
 
-ISSUE: Write this section.
+Like the Storage Access API, {{Document/requestStorageAccessForOrigin(origin)}} is intended to enable removal of cross-site cookies. It enables developers to re-gain cross-site cookies with additional constraints.
+
+Note that many of the same considerations as in <a href="https://privacycg.github.io/storage-access/#privacy">Storage Access API privacy</a> apply. This section primarily covers the differences.
+
+[=requestStorageAccess=] requires interaction with an embedded document. By requiring interaction only with the top-level document, {{Document/requestStorageAccessForOrigin(origin)}} 
+lowers the bar for a potential prompt, though embedded documents can also be quite prominent (or use other techniques to get user interaction).
+[=Implementation-defined=] acceptance and rejection steps are intended to allow user agents to reject abusive requests based on logic they see fit.
+The prompts used have to be careful to indicate the direction of the request, such that the user is able to understand who is requesting access.
+
+As with [=requestStorageAccess=], the same tension between user consent and prompt fatigue exists with {{Document/requestStorageAccessForOrigin(origin)}}; much like the Storage Access API, 
+[=implementation-defined=] acceptance and rejection steps are intended to enable implementers with differing stances on this question to make compromises as they see fit.
 
 <h2 id="security">Security considerations</h2>
 
-ISSUE: Write this section.
+It is important that {{Document/requestStorageAccessForOrigin(origin)}} not degrade security properties of the web platform, even when compared to post-removal of cross-site cookies. 
+Third-party cookie removal has potential benefits for security, specifically in mitigating attacks that rely upon authenticated requests, e.g. CSRF. 
```

Comment:
It might make sense to link to https://docs.google.com/document/d/1AsrETl-7XvnZNbG81Zy9BcZfKbqACQYBSrjM3VsIpjY/edit here? We should maybe also do that in the SAA spec.